src/components/placeBidForm.tsx

```diff
@@ -0,0 +1,134 @@
+import 'react-datepicker/dist/react-datepicker.css';
+
+import { addDoc, collection, doc, getDoc } from 'firebase/firestore';
+import { auth, db, storage } from '../firebase';
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form';
+import { ref, uploadBytes } from "firebase/storage";
+import { useEffect, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+
+
+
+interface propsType{
+    requestId: string,
+    userId: string
+}
```

Comment:
Add a folder containing this component and interface file containing the interfaces used in the component